src/OrchardCore/OrchardCore/Modules/ModularTenantRouterMiddleware.cs

```diff
@@ -16,7 +16,7 @@
 namespace OrchardCore.Modules
 {
     /// <summary>
-    /// Handles a request by forwarding it to the tenant specific <see cref="IRouter"/> instance.
+    /// Handles a request by forwarding it to the tenant specific instance.
```

Comment:
Maybe => `to the tenant specific pipeline`

Reply:
Done